padd.sh

```diff
@@ -1128,36 +1212,89 @@ DisplayHelp() {
 ::: Note: If no option is passed, then stats are displayed on screen, updated every 5 seconds
 :::
 ::: Options:
+:::  -xoff [num]  set the x-offset, reference is the upper left corner
+:::  -yoff [num]  set the y-offset, reference is the upper left corner
 :::  -j, --json    output stats as JSON formatted string
 :::  -h, --help    display this help text
 EOM
     exit 0
 }
 
-if [ $# = 0 ]; then
-  # Turns off the cursor
-  # (From Pull request #8 https://github.com/jpmck/PADD/pull/8)
-  setterm -cursor off
-  trap '{ setterm -cursor on ; echo "" ; exit 0 ; }' INT TERM EXIT
+setOffsets(){
+    # sets x/y-offsets (for shifting the Dashboard within the screen) if CLI arguments were passed
+    i=1;
+    j=$#;
+    while [ $i -le $j ]
+    do
+        case "$1" in
+            "-xoff"  ) xOffset=$2; xOffOrig=$2;;
+            "-yoff"  ) yOffset=$2; yOffOrig=$2;;
+        esac
+        i=$((i + 1));
+        shift 1;
+    done
+}
```

Comment:
You don't need to use these 2 auxiliary variables. 
`shift` will automatically reduce the value of `$#` until there is no more parameters.

```suggestion
setOffsets(){
    # sets x/y-offsets (for shifting the Dashboard within the screen) if CLI arguments were passed
    while [ $# -gt 0 ]
    do
        case "$1" in
            "-xoff"  ) xOffset="$2";;
            "-yoff"  ) yOffset="$2";;
        esac
        shift 2;
    done
}
```

Reply:
@rdwebdesign We still need `xOffOrig=$2` and `yOffOrig=$2` to restore offsets after terminals have been made narrower and then wider again. See https://github.com/pi-hole/PADD/pull/268/commits/47d28c2a6bc671abf5beb566eebbed69218eba39

Reply:
I edited my original post... the variables are here.